.github/workflows/unit-test.yml

```diff
@@ -120,7 +120,7 @@ jobs:
         uses: golangci/golangci-lint-action@v3.6.0
         with:
           version: latest
-          args: --timeout=10m --whole-files -v
```

Comment:
I think we need to see which check takes the longest and maybe we should consider moving it post-merge